x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsRepositoryIntegTests.java

```diff
@@ -110,4 +117,130 @@ public void testRepositoryUsedBySearchableSnapshotCanBeUpdatedButNotUnregistered
 
         assertAcked(clusterAdmin().prepareDeleteRepository(updatedRepositoryName));
     }
+
+    public void testMountIndexWithDeletionOfSnapshotFailsIfNotSingleIndexSnapshot() throws Exception {
```

Comment:
Can we add a test that demonstrates that restoring a searchable snapshot cannot be done if its `delete_searchable_snapshot` flag is in conflict with existing searchable snapshots?

Reply:
Sure, I pushed more tests in ff0f8340ca72d80c76bffb5aadb17aba22d87a1a